CHANGELOG.md

```diff
@@ -439,7 +439,7 @@ section [#1969](https://github.com/plotly/plotly.py/pull/1969).
  - The width of a figure produced by the `create_gantt` figure factory now resizes responsively ([#1724](https://github.com/plotly/plotly.py/pull/1724))
 
 ### Fixed
- - The name of the steps property of `graph_objects.indicator.Guage` has been renamed from `stepss` to `steps`
+ - The name of the steps property of `graph_objects.indicator.Gauge` has been renamed from `stepss` to `steps`
```

Comment:
I'm very impressed that the spell-checking finds this mistake but ignores the `stepss` on the same line! How does this work? Are you manually reviewing the changes before submitting the PRs?

Reply:
Appreciate the comments @nicolaskruchten. No advanced NLP used here, but very basic NLP stuff. Firstly I have a corpus of english words with me. In my second step I create a corpus from all the documents in the project (lets call this plotly corpus). Then I count all the words in the plotly corpus. Then I add all words in plotly corpus having word count greater than a fixed threshold to the english ones. This step ensures that the programming jargon is not counted as a spelling error.

Now all words below the threshold are the errors. If any document contains these words, my script just prints out the filename and the errored word. I go through this and fix them manually.

Reply:
Interesting! `stepss` is definitely not anywhere else, or shouldn't be. Its inclusion in this line of the changelog is specifically to point out that we made a mistake and had to fix it ;)

Either way, thanks again for the contribution :)

Reply:
There is lot of manual work involved. I go through the present sentence understand the context and then read the correction suggestion and take my call.